influunt-api/app/checks/AoMenosUmGrupoSemaforicoValidator.java

```diff
@@ -16,7 +16,7 @@ public void initialize(AoMenosUmGrupoSemaforico constraintAnnotation) {
 
     @Override
     public boolean isValid(Anel anel, ConstraintValidatorContext context) {
-        if (anel.isAtivo()) {
+        if (anel.getGruposSemaforicos() != null && anel.isAtivo()) {
```

Comment:
Em qual situação que um anel ativo não vai ter grupos semafóricos? Depois de passar da tela de anéis e antes de chegar na associação de grupos semafóricos?